### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by shinevue**

### CODE REVIEW

**Review by Johnhvy — APPROVED** [review]
Ok.
No problem.

**Review by chivalrousdev — APPROVED** [review]
Ok

**Comment by creative2113** [comment]
a little poor.

**Review by techietrend — APPROVED** [review]
I am approving

**Comment by BlackGhost2693** [comment]
ok, good idea

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+shine401020@gmail.com.
```
Comment (Ghost1215) [inline CODE_OF_CONDUCT.md:63]: It prefer to be noticed from the client.
It is good for you.

Comment (Ghost1215) [reply CODE_OF_CONDUCT.md:63]: cool.
That is a great idea.

File: CODE_OF_CONDUCT.md
Comment (Ghost1215) [inline CODE_OF_CONDUCT.md:1]: cool.

**Review by Ghost1215 — COMMENTED** [review]
cool

File: CODE_OF_CONDUCT.md
Comment (joellake992) [reply CODE_OF_CONDUCT.md:1]: Hey, approve this code
Don't left conversation.

**Review by joellake992 — APPROVED** [review]
I solved conversation and agree you

**Pull request merged**